client/package-lock.json

```diff
@@ -5487,34 +5519,6 @@
       "resolved": "https://registry.npmjs.org/boolbase/-/boolbase-1.0.0.tgz",
       "integrity": "sha1-aN/1++YMUes3cl6p4+0xDcwed24="
     },
-    "boxicons": {
-      "version": "2.0.9",
-      "resolved": "https://registry.npmjs.org/boxicons/-/boxicons-2.0.9.tgz",
-      "integrity": "sha512-nsXBqNgtEtEu/TYuOlH4mZ8sdM4ol1/6q4Sb9i0SubhOxbW9SqDemaN1D4yZHC8TfMWGM2tIW0y4YWbDMmuBYg==",
-      "requires": {
-        "@webcomponents/webcomponentsjs": "^2.0.2",
-        "prop-types": "^15.6.0",
-        "react": "^16.0.0",
-        "react-dom": "^16.0.0",
-        "react-interactive": "^0.8.1",
-        "react-router-dom": "^4.2.2"
-      },
-      "dependencies": {
-        "react-router-dom": {
-          "version": "4.3.1",
-          "resolved": "https://registry.npmjs.org/react-router-dom/-/react-router-dom-4.3.1.tgz",
-          "integrity": "sha512-c/MlywfxDdCp7EnB7YfPMOfMD3tOtIjrQlj/CKfNMBxdmpJP8xcz5P/UAFn3JbnQCNUxsHyVVqllF9LhgVyFCA==",
-          "requires": {
-            "history": "^4.7.2",
-            "invariant": "^2.2.4",
-            "loose-envify": "^1.3.1",
-            "prop-types": "^15.6.1",
-            "react-router": "^4.3.1",
-            "warning": "^4.0.1"
-          }
-        }
-      }
-    },
```

Comment:
Update: `boxicons` requires `react-router` which was why it was imported inside

Reply:
That's already the case, no?